# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/OffscreenCanvas.h
# Commit: 851d0144fd91
# Full Hash: 851d0144fd916102f9570ea563a2d91946c759fe
# Author: Jens Stutte <jstutte@mozilla.com>
# Date: 2022-06-23 03:54:50
# Regressor Bug: 1769878
# File Overlap Count: 1
# Description:
#   Bug 1769878 - Remove redundant RefPtr<nsIGlobalObject> from EncodeComplete callback. r=smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147858
# ==============================================================================

diff -r b202476826d2 -r 851d0144fd91 dom/canvas/OffscreenCanvas.h
--- a/dom/canvas/OffscreenCanvas.h	Wed Jun 22 14:06:30 2022 +0000
+++ b/dom/canvas/OffscreenCanvas.h	Wed Jun 22 14:06:30 2022 +0000
@@ -160,7 +160,7 @@
   ~OffscreenCanvas();
 
   already_AddRefed<EncodeCompleteCallback> CreateEncodeCompleteCallback(
-      nsCOMPtr<nsIGlobalObject>&& aGlobal, Promise* aPromise);
+      Promise* aPromise);
 
   void CanvasAttrChanged() {
     ErrorResult dummy;
